templates/typescript_gapic/_util.njk

```diff
@@ -74,8 +74,12 @@ limitations under the License.
 
 
 {%- macro printPageAsyncHeader(method) %}
+{%- if method.pagingMapResponseType -%}
+{{ printCommentsForMethod(method) }}
```

Comment:
Is the idea that for the methods where we say they're equivalent to some other method, we don't repeat the method comments (ie don't call  `printCommentsForMethod`), but in method where we don't refer to a another methods we do insert the original comment? (Just making sure I understand why this call is in the `if` and not in the `else`).

Reply:
Exactly.

`printCommentsForMethod(method)` => is printing the comments above the RPC definition in protos(.i.e [aggregatedList](https://github.com/googleapis/googleapis/blob/master/google/cloud/compute/v1small/compute_small.proto#L671)). It only print for diregapic LRO with map response. Otherwise, we say "equivalent to some other method".